pages/1.10/security/ent/tls-ssl/ca-custom/index.md

```diff
@@ -8,26 +8,322 @@ excerpt:
-1. Before installing DC/OS to all nodes, place the private key file corresponding to the custom CA certificate specified in `ca_certificate_key_path` securely on all master nodes in `/var/lib/dcos/pki/tls/CA/private/custom_ca.key`. The file must be readable by the root user and have 0600 permissions set.
+The private key in the PKCS#8 format associated with the custom CA certificate.
+
+- Certificate chain associated with the custom CA certificate
+
+The complete CA certification chain required for end-entity certificate verification. It has to include the certificates of all parent CAs of the intermediate custom CA up to and including the self-signed root CA certificate. Empty, if the custom CA certificate is a self-signed root CA certificate.
+
+- Configuration
+
+The set of the configuration parameters that governs the specific aspects of the installation procedure. The configuration is stored in the configuration file.
+
+- DC/OS Configuration file
+
+The file which contains the DC/OS configuration parameters, in particular, the  filesystem relative paths to the custom CA certificate, associated private key and the certificate chain. 
+
+The configuration file is normally called `config.yaml` and has to be present in the `genconf/` directory on the bootstrap node during the installation. It is used by the DC/OS installer to create a custom DC/OS build.
+
+- Installation directory
+
+The directory on the bootstrap node where the DC/OS installer resides. It contains the `genconf/` subdirectory where the configuration file, the custom CA certificate, the associated private key and the certificate chains are put. It is denoted with `$DCOS_INSTALL_DIR` in this document. This is the current working directory when executing `dcos_generate_config.ee.sh`.
+
+# Requirements
+
+- Advanced installation method
+In order to install DC/OS Enterprise with a custom CA certificate you have to use the [advanced DC/OS installation method](/1.10/installing/custom/advanced/). Other installation methods are not supported.
+
+- Custom CA certificate
+- Private key associated with the custom CA certificate
+- Certificate chain associated with the custom CA certificate
+
+The files containing the custom CA certificate, the associated private key and the certificate chain has to be placed in the predefined locations where the installer can find them during the installation. (see the next sections for details)
+
+
+## Manually placing the the custom CA certificate, the associated private key and the certificate chain onto the bootstrap node
+The custom CA certificate, the associated private key and the certificate chain files have to be put in the `genconf/` directory on the bootstrap node. The paths to these files have to be specified in the configuration file.
+
```

Comment:
Not really happy with this "action" being a "requirement". I think this should also be combined with 

```
+```bash
 +cd $DCOS_INSTALL_DIR
 +ls genconf/
 +```
 +```bash 
 +dcos-ca-certificate.crt
 +dcos-ca-certificate-key.key
 +dcos-ca-certificate-chain.crt
 +```
```

below.

Reply:
When they read

> The custom CA certificate, the associated private key and the certificate chain files have to be put in the `genconf/` directory on the bootstrap node.

they will be a little confused if this is now something they should do right now, or if this is going to be explained in more detail below.

I think this should be a more specific call to action with the example that's actually given further below, and the vague "The paths to these files have to be specified in the configuration file." should go away (is a more specific call to action further below).

(this is the stuff I only notice when looking at a beautifully rendered document, and that I didn't really watch out before).

Reply:
ok, I have moved the 'ls' command from the "Installing DC/OS Enterprise with a custom CA certificate", "Starting point" subsection to this section. (combined it with the "The custom CA certificate, the associated private key and the certificate chain files have to be put in the `genconf/` directory on the bootstrap node." sentence)
If this is not what you meant please leave a comment.